app/src/main/java/net/squanchy/search/SearchActivity.kt

```diff
@@ -193,6 +189,11 @@ class SearchActivity : AppCompatActivity(), SearchRecyclerView.OnSearchResultCli
         }
     }
 
+    override fun onSaveInstanceState(outState: Bundle?) {
+        outState?.putString(QUERY_KEY, searchField.text?.toString())
```

Comment:
⚠️  Note that if the `outState` is null it's your responsibility to create it here and pass it down!

Reply:
Are you sure about this? I think it's actually not null, it's just lacking the annotation as some other support library code. We use it also in the HomeActivity where we consider the `Bundle` not null

Reply:
Google Kotlin samples leave it as nullable... but yeah they should really have annotations